### PR TITLE
[shinyapps.io] Add ability to resend invitations 

### LINKED ISSUE #543: Get pending invitation links from the library
We can currently invite users to an app using the the library, but, AFAIK, it is not possible to retrieve the invitation link after that. In case a user is already invited, we are forced to go to the web console manually to get the invitation link.

It would be REALLY appreciated to be allowed to retrieve invitation links from the library.

### PR DESCRIPTION
**Pull request by kippandrew**
Resolves #543 

This PR adds a new function `resendInvitation` that will resend an invitation email (and optionally regenerate the invitation code). An invites expiration status has also been added to the dataframe returned by `showInvited`.

### CODE REVIEW

File: R/auth.R [hunk 1]
```diff
@@ -220,6 +221,56 @@ showInvited <- function(appDir=getwd(), appName=NULL, account = NULL,
...
+                             appDir=getwd(), appName=NULL,
+                             account = NULL, server=NULL) {
+
+  # resolve account
+  accountDetails <- accountInfo(resolveAccount(account, server), server)
+
+  # get invitations
+  invited <- showInvited(appDir, appName, account, server)
+
+  if (is.numeric(invite)) {
```
Comment (aronatkins) [inline R/auth.R:249]: Is there any point to handling a vector of invitations?

**Review by aronatkins — APPROVED** [review]

File: R/auth.R (hunk 1 above)
Comment (kippandrew) [reply R/auth.R:249]: Potentially useful, but it would be inconsistent with the other APIs. 🤷‍♂️

Comment (aronatkins) [reply R/auth.R:249]: kk. as-is it is, then! let me know once you're happy with testing and we'll get this merged.

Comment (kippandrew) [reply R/auth.R:249]: Any idea why Travis is complaining? The output looks like CRAN check non-sense to me. Are these transient failures?

Comment (aronatkins) [reply R/auth.R:249]: No clue, sadly, and other recent PRs passed.

Comment (aronatkins) [reply R/auth.R:249]: When in doubt, regenereate! 🎉

Comment (kippandrew) [reply R/auth.R:249]: I think the travis output is being truncated, but I found this running it locally.

**Pull request merged**